glib/src/auto/functions.rs

```diff
@@ -823,6 +837,26 @@ pub fn spawn_command_line_async(
 //    unsafe { TODO: call ffi:g_spawn_sync() }
 //}
 
+#[cfg(any(feature = "v2_74", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v2_74")))]
+#[doc(alias = "g_timeout_add_once")]
+pub fn timeout_add_once<P: Fn() + Send + Sync + 'static>(interval: u32, function: P) -> u32 {
```

Comment:
Needs to be implemented manually, see how things are done for `g_idle_add()` in `src/source.rs`